recipes/plio/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "plio" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/USGS-Astrogeology/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: a189e7ee6973a209aee81ca412144599fdf7d2aefbb03c1d93c37212be053a7b
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
```

Comment:
This should be called `host` and should only contain packages needed to install the code, not run it. This is any package imported in the setup.py plus pip and python.